templates/workspace_services/guacamole/user_resources/guacamole-azure-linuxvm/porter.yaml

```diff
@@ -1,7 +1,7 @@
 ---
 schemaVersion: 1.0.0
 name: tre-service-guacamole-linuxvm
-version: 0.6.9
+version: 0.6.17
```

Comment:
I am not sure if semantic versioning is used for these bundles, but it feels like this should be at least a minor version increase, if not a major.
(E.g., how will the Update command work?)

Reply:
agreed will change then to 1.0.0 when I've finished testing